eng/versioning/version_client.txt

```diff
@@ -54,7 +54,7 @@ com.azure:azure-communication-identity;1.0.0-beta.6;1.0.0
 com.azure:azure-communication-phonenumbers;1.0.0-beta.6;1.0.0-beta.7
 com.azure:azure-containers-containerregistry;1.0.0.beta.1;1.0.0-beta.1
 com.azure:azure-core;1.15.0;1.16.0-beta.1
-com.azure:azure-core-amqp;2.0.3;2.1.0-beta.1
+com.azure:azure-core-amqp;2.0.3;2.2.0-beta.1
```

Comment:
Should be `2.1.0-beta.2`, no need to increment the minor version after a beta release